lib/internal/Magento/Framework/View/Page/Config/Renderer.php

```diff
@@ -136,6 +136,12 @@ public function renderMetadata()
     protected function processMetadataContent($name, $content)
     {
         $method = 'get' . $this->string->upperCaseWords($name, '_', '');
+        if($name === 'title') {
+            if (!$content) {
+                $content = $this->escaper->escapeHtml($this->pageConfig->$method()->get());
```

Comment:
Why do we need `escapeHtml` here? Seems inconsistent with other meta data getters.

Maybe just add `__toString` to `\Magento\Framework\View\Page\Title`?

Reply:
Becaouse in the same file at line [109](https://github.com/slackerzz/magento2/blob/32e2e4de5921ecb7b13722955298e886ec2b2d59/lib/internal/Magento/Framework/View/Page/Config/Renderer.php#L109) we have:

`return '<title>' . $this->escaper->escapeHtml($this->pageConfig->getTitle()->get()) . '</title>' . "\n";`

so i reproduced the same behaviour

Reply:
I think the original reason for that is that if someone enter some html inside the `title` is has to be escaped to not break the html